docs/organizations/overview.mdx

```diff
@@ -5,26 +5,150 @@ description: Learn about how to create and manage Clerk organizations and their
+
+You can also use the `setActive()` method, which is available on the [`Clerk`](/docs/references/javascript/clerk/session-methods#set-active) object and is returned by the [`useOrganizationList()`](/docs/references/react/use-organization-list) hook.
+
+## Create an organization
+
+### Application owner
+
+You can create organizations in the Clerk Dashboard or in your application. To create an organization in the Clerk Dashboard:
+
+1. Navigate to the [Clerk Dashboard](https://dashboard.clerk.com/last-active?path=organizations).
+2. In the navigation sidebar, select **Organizations**.
+3. Select the **Create Organization** button.
+4. Enter the organization's name and slug. The slug is a unique identifier for the organization and is used in URLs. Select the organization's owner from the list of users in your application. The owner is the user who will be the organization's admin.
+
+#### Monthly active organization (MAO)
+
+The number of organizations you can have in a single Clerk application depends on your [Clerk plan](https://clerk.com/pricing) and the type of instance (development or production), and is measured by "monthly active organizations" (MAOs). An MAO is an organization with more than one user that has signed in that month.
+
+With the Free plan:
+- In development instances, you can have *up to* 50 MAOs in a single Clerk application. Each MAO can have *up to* 5 members.
+- In production instances, you can have up to 100 MAOs in a single Clerk application. Each MAO can have up to 5 members.
+
+With the Pro plan:
+- In development instances, you can have an unlimited number of MAOs in a single Clerk application *for free*. Each MAO can have an unlimited number of members.
+- In production instances, you can have up to 100 MAOs in a single Clerk application *for free*. Each MAO after the first 100 costs $1.00 per month. Each MAO can have an unlimited number of members.
+
+For more details on pricing, see [our pricing page](https://clerk.com/pricing).
+
+If you need more organizations or custom pricing, please contact [our sales team](https://clerk.com/contact/sales) to upgrade to the Enterprise plan.
+
+### Application user
+
+Users can create organizations within your application. When a user creates an organization, they become the organization's admin. As the organization's admin, they have full control over the organization, including the ability to update the organization's settings, invite users to join the organization, and manage the organization's members.
+
+A user within one of your applications can create *up to* 100 organizations in that application. However, the number of [MAOs](#monthly-active-organization-mao) for that application must be kept in mind.
+
+The easiest way to allow users to create organizations is to use Clerk's [`<CreateOrganization />`](/docs/components/organization/create-organization) and/or [`<OrganizationSwitcher />`](/docs/components/organization/organization-switcher) components.
```

Comment:
I don't see this setting anywhere in the Dashboard, so what exactly would I be referencing to ?
"By default, all users have the ability to create organizations. If you would like to change this, you can add a custom permission... " something like that ??

Reply:
<img width="1063" alt="Screenshot 2024-04-15 at 4 33 35 PM" src="https://github.com/clerk/clerk-docs/assets/556932/5f28a6f9-9b59-43e0-abbe-ed5e64d26510">

It can be found here, kinda buried. It can also be toggled on a per-user basis. The "defaulting" behavior is a bit unintuitive, and is the same as the ability to delete

Reply:
just added the info !